sklearn/model_selection/tests/test_split.py

```diff
@@ -368,24 +369,56 @@ def test_stratified_kfold_no_shuffle():
         list(StratifiedKFold(2).split(X, y1)),
         list(StratifiedKFold(2).split(X, y2)))
 
+    # Check equivalence to KFold
+    y = [0, 1, 0, 1, 0, 1, 0, 1]
+    X = np.ones_like(y)
+    np.testing.assert_equal(
+        list(StratifiedKFold(3).split(X, y)),
+        list(KFold(3).split(X, y)))
+
 
-def test_stratified_kfold_ratios():
+@pytest.mark.parametrize('shuffle', [False, True])
+@pytest.mark.parametrize('k', [4, 5, 6, 7, 8, 9, 10])
```

Comment:
reduce number of k maybe?

Reply:
I figure it's a pretty quick test, and I found some implementation bugs appeared only on some K...